client/components/happychat/notices.jsx

```diff
@@ -16,22 +16,29 @@ import {
 	HAPPYCHAT_CHAT_STATUS_ABANDONED,
 	getHappychatStatus,
 	getHappychatConnectionStatus,
+	isHappychatServerReachable,
 } from 'state/happychat/selectors';
 
 /*
  * Renders any notices about the chat session to the user
  */
 class Notices extends Component {
 	statusNotice() {
-		const { connectionStatus, chatStatus, translate } = this.props;
+		const { isServerReachable, connectionStatus, chatStatus, translate } = this.props;
+
+		if ( ! isServerReachable ) {
+			return translate( "We're having trouble connecting to chat. Please check your internet connection while we try to reconnect…" );
+		}
 
 		switch ( connectionStatus ) {
```

Comment:
Thanks, this is on my to-do list before I move off the middleware cleanup tasks, but I didn't want to pollute existing PRs with that stuff.